suspend-org.sh

```diff
@@ -10,11 +10,15 @@ if [ "$#" -ne 1 ]; then
     exit 1
 fi
 
-GUID=$(cf org $1 --guid)
+org_name=$1
+GUID=$(cf org "${org_name}" --guid)
 
-cf curl /v2/organizations/$GUID -X PUT -d '{"status":"suspended"}'
+cf curl "/v2/organizations/${GUID}" -X PUT -d '{"status":"suspended"}'
 
-for route in $(cf curl /v2/routes?q=organization_guid:$GUID | jq -r .resources[].metadata.guid)
-do
-  cf curl -X DELETE /v2/routes/$route?recursive=true
```

Comment:
We don't actually want to run any deletes on anything when we do this, because we always want to try to unsuspend things in the future. See :eyes: 18F/cg-site#1380